docs/ai_actions/ai_actions_guide.md

```diff
@@ -8,7 +8,7 @@ month_change: false
 ## What are AI Actions
 
 Wherever you look, artificial intelligence becomes more and more important by enhancing user interaction and automating complex processes.
-[[= product_name =]] can be equipped with the AI Actions [LTS update](ibexa_dxp_v4.6.md#lts-updates), which harnesses AI's potential to automate time-consuming editorial tasks.
+[[= product_name =]] can be equipped with the AI Actions [LTS update](editions#lts-updates), which harnesses AI's potential to automate time-consuming editorial tasks.
```

Comment:
> `INFO    -  Doc file 'ai_actions/ai_actions_guide.md' contains an unrecognized relative link 'editions#lts-updates', it was left as is.`

The file extension is missing, it should have been `editions.md#lts-updates`

> `INFO    -  Doc file 'ai_actions/ai_actions_guide.md' contains a link '../release_notes/ibexa_dxp_v4.6.md#lts-updates', but the doc 'release_notes/ibexa_dxp_v4.6.md' does not contain an anchor '#lts-updates'.`

And there was [another one line 28](https://github.com/ibexa/documentation-developer/blob/6748eb397a29a2f64c370e574c2828516caea929/docs/ai_actions/ai_actions_guide.md?plain=1#L28)

Fixed in #2686